android/src/main/java/com/polidea/blemulator/SimulatedAdapter.java

```diff
@@ -324,7 +324,7 @@ public void onSuccess(DeviceContainer deviceContainer) {
                     if (service.getId() == serviceIdentifier) {
                         return deviceContainer
                                 .getCharacteristics()
-                                .get(service.getUuid().toString().toUpperCase())
+                                .get(service.getUuid().toString().toUpperCase()) //HERE; TODO remove mark
```

Comment:
Is it on purpose or you forgot to commit the fix?

Reply:
Forgot to remove the comment from analysis. Fix is in DartMethodCaller.

Reply:
Forgot to remove the comment from analysis. Fix is in DartMethodCaller.